.env.dist

```diff
@@ -20,6 +20,9 @@ DOCTRINE_REPORT_DATABASE=gewisdb_report
 APP_ENV=production
 COOKIE_DOMAIN=localhost
 
+# Public domain for emails
+APP_URL=https://database.gewis.nl
```

Comment:
This is the only part I do not like about the e-mail template but unfortunately there is no other way to get `serverUrl()` code into the template without executing it in the service.

Reply:
Still stands, but resolving to clear blockages.